stylesheet/style.css

```diff
@@ -0,0 +1,86 @@
+body{
+  background-color: #EAEAEA;
+}
+.header-container{
+  display: flex;
+flex-direction: column;
+flex-wrap: wrap;
+}
+ li{
+  display: inline;
+  font-size: 1.2rem;
+}
+.logo-bar {
+  background-color: #093C60;
+}
+.image-bar img{
+  height: 200px;
+  flex:2;
+}
+.logo-bar img{
+  height: 200px;
+  flex:1;
+}
+.main-container{
+display: flex;
+background-color: white;
+}
+.dogs-container{
+  display: flex;
+flex-wrap: wrap;
+justify-content:space-between;
+background-color: white;
+}
+.blog-container{
+  display: flex;
+  flex-direction: row;
+justify-content:space-between;
+background-color: white;
+width:100%;
+}
+.main-container div, .dogs-container div , .blog-container div{
+  padding:10px;
+}
+.box-1
```

Comment:
I'd use class names that are more semantic, like `box-image-wrapper` or something.